src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -85,6 +87,7 @@ public Task StartAsync(CancellationToken cancellationToken)
 
         public Task StopAsync(CancellationToken cancellationToken)
         {
+            StopFileWatchers();
```

Comment:
This change highlighted an interesting scenario -- when we stop the WebHost, it will also stop the running JobHost. However, if that JobHost.Stop() call takes too long (currently 10 seconds), we may not ever actually *dispose* the JobHost. This means that some services may take WebHost dependencies... which get disposed before the JobHost.

Everything is shutting down at that point so it may not matter, but in this scenario I wanted to try to stop these watchers when we're stopping the host -- I think it makes logical sense anyway.